packages/transition-frontend/src/components/forms/comparison/ScenarioComparisonAlternativesSelect.tsx

```diff
@@ -0,0 +1,202 @@
+ */
+import React, { useState } from 'react';
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons/faAngleRight';
+import { faAngleLeft } from '@fortawesome/free-solid-svg-icons/faAngleLeft';
+
+import ScenarioComparisonResults from './ScenarioComparisonResults';
+import Button from 'chaire-lib-frontend/lib/components/input/Button';
+import { RoutingResult } from 'chaire-lib-common/lib/services/routing/RoutingResult';
+import serviceLocator from 'chaire-lib-common/lib/utils/ServiceLocator';
+import { default as FormErrors } from 'chaire-lib-frontend/lib/components/pageParts/FormErrors';
+import { TransitRoutingAttributes } from 'transition-common/lib/services/transitRouting/TransitRouting';
+import { EventManager } from 'chaire-lib-common/lib/services/events/EventManager';
+import { MapUpdateLayerEventType } from 'chaire-lib-frontend/lib/services/map/events/MapEventsCallbacks';
+import { SegmentToGeoJSONFromPaths } from 'transition-common/lib/services/transitRouting/TransitRoutingResult';
+import { useTranslation } from 'react-i18next';
+import { TrRoutingRoute } from 'chaire-lib-common/lib/services/transitRouting/types';
+
+export interface AlternativesSelectProps {
+    result1: RoutingResult;
+    result2: RoutingResult;
+    request: TransitRoutingAttributes;
+    scenarioNames: {
+        name1: string;
+        name2: string;
+    };
+    hasAlternativeWalkPath: {
+        result1: boolean;
+        result2: boolean;
+    };
+}
+
+export interface AlternativesSelectOneScenarioProps {
+    result: RoutingResult;
+    alternativeIndex: number;
+    setAlternativeIndex: (arg: number) => void;
+    useAlternateLayerAndColor: boolean;
+}
+
+const showCurrentAlternative = async (result, alternativeIndex, useAlternateLayer) => {
+    const pathCollection = serviceLocator.collectionManager.get('paths');
+    const segmentToGeojson = new SegmentToGeoJSONFromPaths(pathCollection);
+    const options = { completeData: false, segmentToGeojson: segmentToGeojson.segmentToGeoJSONFromPaths };
+    const pathGeojson = await result.getPathGeojson(alternativeIndex, options);
+    (serviceLocator.eventManager as EventManager).emitEvent<MapUpdateLayerEventType>('map.updateLayer', {
+        layerName: 'routingPoints',
+        data: result.originDestinationToGeojson()
+    });
+    if (useAlternateLayer) {
+        serviceLocator.eventManager.emit('map.updateLayers', {
+            routingPathsAlternate: pathGeojson,
+            routingPathsStrokesAlternate: pathGeojson
+        });
+    } else {
+        serviceLocator.eventManager.emit('map.updateLayers', {
+            routingPaths: pathGeojson,
+            routingPathsStrokes: pathGeojson
+        });
+    }
+};
+
+const AlternativesSelect: React.FunctionComponent<AlternativesSelectProps> = (props: AlternativesSelectProps) => {
+    const { t } = useTranslation(['transit']);
+
+    const [alternativeIndex1, setAlternativeIndex1] = useState(0);
```

Comment:
Just checking, it's normal that both have the same useState?? (Maybe we need a small comment here)